Makefile

```diff
@@ -746,7 +746,7 @@ endif
 roxvet: $(ROXVET_BIN)
 	@echo "+ $@"
 	@# TODO(ROX-7574): Add options to ignore specific files or paths in roxvet
-	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./... | grep -v 'operator/pkg/clientset')
+	$(SILENT)go list -e ./... | grep -v 'operator/pkg/clientset' | xargs -n 1000 go vet -vettool "$(ROXVET_BIN)"
```

Comment:
See https://github.com/stackrox/stackrox/pull/8611 for an alternative fix. 

My concern with this solution is that if I understand it correctly, some of the files will be cut off from `roxvet` command, meaning it would be possible to merge improperly styled filed.

Reply:
Indeed, I think I misread the `xargs` man page.
I'd rather split the input per directory.
I'm not sure what difference there is between `$(cat tmpfile)` and `$(go list -e ./...)`.